server/controllers/shared/referralsController.test.ts

```diff
@@ -91,6 +91,23 @@ describe('ReferralsController', () => {
         submittedText: `Submitted in referral on ${DateUtils.govukFormattedFullDateString(referral.submittedOn)}.`,
       })
     })
+
+    describe('when the referral has not been submitted', () => {
+      it('redirects to the new referral additional information page', async () => {
```

Comment:
I think we should 403 here rather than redirecting, in future, only the person who created the referral should be able to send it off. It's unlikely they'd get to this page in the usual user journey.

Reply:
400ing as discussed